src/server/cluster/cluster_family.cc

```diff
@@ -79,6 +79,21 @@ ClusterConfig* ClusterFamily::cluster_config() {
   return tl_cluster_config.get();
 }
 
+void ClusterFamily::Shutdown() {
+  shard_set->pool()->at(0)->Await([this] {
```

Comment:
why at 0?

Reply:
I didn't like this line either, but the way we initialize shards, 0..n are always shards, n..m can be connection threads

Reply:
Why should we run this command on the shards? I thought just run it in the same thread